etc/validation-error-message-formatter.js

```diff
@@ -44,14 +44,34 @@ exports.maximumValueViolation = function(itemPath, maxValue) {
   return 'item "' + itemPath + '" must not be greater than ' + maxValue;
 };
 
+/**
+ * Formats a message for the error that occurs when a value is greater than or equal to the maximum allowed.
+ *
+ * @param {string} itemPath The full path of the property or element in which the error occurs (e.g. "objectProp.dateProp")
+ * @param {(float|integer|string)} maxValue The maximum value that is allowed
+ */
```

Comment:
You'll find that documentation for all the other formatter functions has been defined in https://github.com/Kashoo/synctos/pull/46.